### PR TITLE
Create Stoofvlees.md

### PR DESCRIPTION
**Pull request by Innen**
To create the stoofvlees (NL)

### CODE REVIEW

File: Stoofvlees.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+### Ingredients
+- 2 grote uien
+- klontjes boter
+- 1 kg rundsvlees (zenuwstuk, in gelijke stukken)
+- peper van de molen
```
Comment (LilyAnderssonLee) [inline Stoofvlees.md:5]: What is this?

File: Stoofvlees.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- snuifjes zout
+- 2 flesjes bruin bier bv. Sint Bernardus abt 12
+- 2 blaadjes laurier
+- 2 takjes tijm
```
Comment (LilyAnderssonLee) [inline Stoofvlees.md:9]: Can you explain it a little bit more?

**Review by LilyAnderssonLee — APPROVED** [review]
Great recipe, but i don't understand!!!

File: Stoofvlees.md (hunk 1 above)
Comment (Innen) [reply Stoofvlees.md:5]: It means it's pepper from the table mill.

**Pull request merged**